tests/test_simpleflow/swf/test_executor.py

```diff
@@ -119,3 +125,18 @@ def test_get_event_details(self):
         })
         details = executor.get_event_details('marker', 'another_marker')
         expect(details).to.be.none
+
+        details = executor.get_event_details('timer', 'a_timer')
+        del details['started_event_timestamp']
+        del details['fired_event_timestamp']
+        expect(details).to.equal({
+            'type': 'timer',
+            'state': 'fired',
+            'id': 'a_timer',
+            'start_to_fire_timeout': 1,
+            'started_event_id': 6,
+            'fired_event_id': 7,
+            'control': None,
+        })
+        details = executor.get_event_details('timer', 'another_timer')
+        expect(details).to.be.none
```

Comment:
I like it; the only downside is that PyCharm doesn't do completion, so I need a browser tab open :slightly_smiling_face: